test/curve/ankr.js

```diff
@@ -0,0 +1,110 @@
+// Utilities
+const Utils = require("../utilities/Utils.js");
+const { impersonates, setupCoreProtocol, depositVault } = require("../utilities/hh-utils.js");
+
+const addresses = require("../test-config.js");
+const { send } = require("@openzeppelin/test-helpers");
+const BigNumber = require("bignumber.js");
+const IERC20 = artifacts.require("@openzeppelin/contracts/token/ERC20/IERC20.sol:IERC20");
+
+const CRVStrategyAnkrETHMainnet = artifacts.require("CRVStrategyAnkrETHMainnet");
+
+// Test was developed at blockNumber 11913700
+
+// Vanilla Mocha test. Increased compatibility with tools that integrate Mocha.
+describe("Mainnet Curve Ankr", function() {
+  let accounts;
+
+  // external contracts
+  let underlying;
+
+  // external setup
+  let underlyingWhale = "0x2FedC4002A2398ac32d9d43a9B92A5cf16a36304";
+
+  // parties in the protocol
+  let governance;
+  let farmer1;
+
+  // numbers used in tests
+  let farmerBalance;
+
+  // Core protocol contracts
+  let controller;
+  let vault;
+  let strategy;
+
+  let crv = "0xD533a949740bb3306d119CC777fa900bA034cd52";
+  let onx = "0xE0aD1806Fd3E7edF6FF52Fdb822432e847411033";
+  let ankr = "0x8290333ceF9e6D528dD5618Fb97a76f268f3EDD4";
+  let weth = "0xC02aaA39b223FE8D0A0e5C4F27eAD9083C756Cc2";
+
+  async function setupExternalContracts() {
+    underlying = await IERC20.at("0xaA17A236F2bAdc98DDc0Cf999AbB47D47Fc0A6Cf");
+    console.log("Fetching Underlying at: ", underlying.address);
+  }
+
+  async function setupBalance(){
+    let etherGiver = accounts[9];
+    // Give whale some ether to make sure the following actions are good
+    await send.ether(etherGiver, underlyingWhale, "1" + "000000000000000000");
+
+    farmerBalance = await underlying.balanceOf(underlyingWhale);
+    await underlying.transfer(farmer1, farmerBalance, { from: underlyingWhale });
+  }
+
+  before(async function() {
+    governance = "0xf00dD244228F51547f0563e60bCa65a30FBF5f7f";
+    accounts = await web3.eth.getAccounts();
+
+    farmer1 = accounts[1];
+
+    // impersonate accounts
+    await impersonates([governance, underlyingWhale]);
+
+    await setupExternalContracts();
+    [controller, vault, strategy] = await setupCoreProtocol({
+      "existingVaultAddress": null,
+      "strategyArtifact": CRVStrategyAnkrETHMainnet,
+      "liquidation": {"onx": {"sushi": [onx, weth]},
```

Comment:
Since we don't use the key (`"onx"`), should we turn the whole thing into array instead?
```
"liquidation": [
  {"sushi": [onx, weth]},
  {"uni": [crv, weth]},
  {"uni": [ankr, weth]}
]
```

Reply:
I think this gave me a problem having two entries with the name "uni". It only returned the last one. The first one is then overwritten it seems.

Reply:
How about this?
```
"liquidation": [
  ["sushi", [onx, weth]],
  ["uni", [crv, weth]],
  ["uni", [ankr, weth]]
]
```

Reply:
That did not work, but got it working like this: 

"liquidation": [
    {"sushi":[onx, weth]},
    {"uni":[crv, weth]},
    {"uni":[ankr, weth]}
],

It iterates through numerically.